anki/template/template.py

```diff
@@ -200,20 +205,27 @@ def render_unescaped(self, tag_name=None, context=None):
 
     def clozeText(self, txt, ord, type):
         reg = clozeReg
-        if not re.search(reg%ord, txt):
+        if not re.search(reg % ord, txt):
             return ""
+
         def repl(m):
             # replace chosen cloze with type
+            noSound = removeClozeSound(m.group(3))
```

Comment:
But that's two different cards.  only c1 will show on card 1 and only c2 will show on card 2.
You have 2 questions and 2 answers.

You don't want to hear part 2 when you play part 1 and you don't want part 1 when you are looknig at part 2.

And you can put any sound on the answer or the question,  but hearing all the sounds for all the questions and answers when you have a card with one of the questions isn't going to help you. 
It'll make you not do that anymore.   And then you will realize that audio and clozes don't mix. 
Until now.